autoload/himalaya/domain/email.vim

```diff
@@ -275,6 +275,68 @@ function! himalaya#domain#email#delete() abort range
   \})
 endfunction
 
+function! himalaya#domain#email#flags_add() abort range
+  let ids = stridx(bufname('%'), 'Himalaya envelopes') == 0 ? s:get_email_id_under_cursors(a:firstline, a:lastline) : s:id
+  let flags = input(printf("Select flags to add between (Seen, Answered, Flagged, Deleted, Drafts)\n
```

Comment:
This sounds too complicated to me, and does not fit with the other prompts. Plus it does not handle custom flags. As stated in `himalaya flags --help`:

> Existing flags are seen, answered, flagged, deleted, draft. Other flags are considered custom

I would suggest the following:

```suggestion
  let flags = input("Flags to add: ", "", "custom,himalaya#domain#email#flags#complete")`
```

The function `himalaya#domain#email#flags#complete` could complete standard flags `seen`, `answered`, `flagged`, `deleted` and `draft`, see `:h command-completion-custom`.

```vim
function! himalaya#domain#email#flag#complete(ArgLead, CmdLine, CursorPos) abort
  " TODO
endfunction
```